packages/instantsearch-codemods/README.md

```diff
@@ -0,0 +1,32 @@
+## Codemods
+
+These codemods (code transformers) can be ran with [jscodeshift](https://github.com/facebook/jscodeshift).
+
+### `rish-to-ris`
+
+This will replace all imports from `react-instantsearch-hooks` to their `react-instantsearch` equivalent.
+
+```
+npx @codeshift/cli -p instantsearch-codemods#rish-to-ris <path>
```

Comment:
If I'm not mistaken the `-p` flag only allows valid parser values. If you wanted the packages flag it needs to be set in full:

```suggestion
npx @codeshift/cli --packages 'instantsearch-codemods#rish-to-ris' <path>
```

I also put the package+preset name between single quote to prevent a shell like bash/zsh from considering `#` as a special character.